WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/JetpackDropdownMenu.kt

```diff
@@ -60,10 +64,17 @@ fun JetpackDropdownMenu(
                 isMenuVisible = !isMenuVisible
             }
         )
+        val cascadeMenuWidth = 200.dp
         CascadeDropdownMenu(
-            modifier = Modifier.background(MenuColors.itemBackgroundColor()),
+            modifier = Modifier
+                .background(MenuColors.itemBackgroundColor())
+                .width(cascadeMenuWidth),
```

Comment:
💡 I noticed the `CascadeDropdownMenu` already has a `fixedWidth` parameter that defaults to 196dp, maybe we should use that parameter when defining the width for this component instead of passing a `Modifier.height` since I am not 100% sure if it's treated differently by the lib.

Reply:
Good point, done